_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
+
+## Usage
+
+See the following `.yaml` configuration file:
+
+```
+processor:
+    - otel_metrics_raw_processor
+```
+
+## Configuration
+
+You can create detailed representations of histogram buckets and their boundaries. You can control this feature by using the following parameters:
```

Comment:
What is a histogram bucket? Do we have any docs that explain this?

Reply:
We don't have any docs on it. I think it refers to what's described here: https://opentelemetry.io/docs/reference/specification/metrics/data-model/#histogram